README.md

```diff
@@ -1,13 +1,13 @@
-# Collabo Web App
+# collabo-community-app
```

Comment:
**_Review comment 1:_** Please use idea from the previous title @Dkingofcode 
There are no hyphens and each word starts with a capital letter.